### PR TITLE
add noarch (cont'd)

### PR DESCRIPTION
**Pull request by h-vetinari**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

~An attempt to continue~ closes #4, since that one cannot be easily updated anymore by the tickbot. Reproducing the OP below.

Also: @conda-forge-admin, please rerender
_________________

I think this feedstock could be built with noarch.
This means that the package only needs to be built once, drastically reducing CI usage.
See here for more information about building noarch packages.
Before merging this PR make sure:
* [ ] No compiled extensions
* [ ] No post-link or pre-link or pre-unlink scripts
* [ ] No OS specific build scripts
* [ ] No python version specific requirements
* [ ] No skips except for python version. (If the recipe is py3 only, remove skip statement and add version constraint on python)
* [ ] 2to3 is not used
* [ ] Scripts argument in setup.py is not used
* [ ] If entrypoints are in setup.py, they are listed in meta.yaml
* [ ] No activate scripts
* [ ] Not a dependency of conda

Notes and instructions for merging this PR:
1. If any items in the above checklist are not satisfied, please close this PR. Do not merge.
1. Please merge the PR only after the tests have passed.
1. Feel free to push to the bot's branch to update this PR if needed.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,8 +10,9 @@ source:
     sha256: b6df99b2cd1c75d9d342e4335b535789b8da9107ec748212706ef7bbe5c2553b
 
 build:
-    number: 1000
-    script: python setup.py install --single-version-externally-managed --record record.txt
+    noarch: python
+    number: 1
```
Comment (CJ-Wright) [inline recipe/meta.yaml]: This most likely needs to be set to 1001

**Comment by h-vetinari** [comment]
@CJ-Wright 
Thanks for the tip about the build number, but it's still failing. The logs don't provide much insight, I'm afraid. Not sure how to proceed here...

**Review by CJ-Wright — COMMENTED** [review]
Noarch recipes don't have build sections, just host and run.

**Comment by h-vetinari** [comment]
@CJ-Wright 
Still nothing...

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,13 +10,13 @@ source:
...
-    script: python setup.py install --single-version-externally-managed --record record.txt
+    noarch: python
+    number: 1001
+    script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
-    build:
+    host:
         - python
-        - setuptools
```
Comment (CJ-Wright) [inline recipe/meta.yaml:19]: This needs to be pip. see the `No module named pip` error in the logs

**Comment by h-vetinari** [comment]
Thanks a lot @CJ-Wright, this is green now (I rebased to squash all those mini-commits). I haven't double-checked all the bullet points from the tickbot (reproduced in the OP), this is maybe something for @conda-forge/jsmin ?

**Comment by jakirkham** [comment]
Could you please add a “Closes ...” note in the OP so that PR ( https://github.com/conda-forge/jsmin-feedstock/pull/4 ) will be closed out.

**Comment by h-vetinari** [comment]
@jakirkham 
Done

**Comment by ocefpaf** [comment]
This recipe cannot be `noarch` b/c it uses 2to3. Sorry @h-vetinari.

**Pull request closed without merging**